packages/@aws-cdk/aws-location-alpha/lib/place-index.ts

```diff
@@ -74,39 +74,12 @@ export enum IntendedUse {
   STORAGE = 'Storage',
 }
 
-abstract class PlaceIndexBase extends Resource implements IPlaceIndex {
```

Comment:
@mazyu36 - just to double clarify here, you don't forsee adding an additional class anywhere that would need to extend the base class?

Reply:
@sumupitchayan 

I don't think we'll need to add other classes, since there are no resources similar to placeindex.